src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -821,6 +821,21 @@
      */
     String getMSIClientId();
 
+    /**
+     * Sets the Azure Key Vault (AKV) Provider user principal id.
```

Comment:
This is not necessarily AKV user principal ID. Please align the docs with [ODBC connection property documentation.](https://docs.microsoft.com/en-us/sql/connect/odbc/using-always-encrypted-with-the-odbc-driver?view=sql-server-ver15#using-the-azure-key-vault-provider)